### PR TITLE
Add method for calculating Makkink reference evaporation same as KNMI

### PR DESCRIPTION
**Pull request by martinvonk**
The Royal Netherlands Meteorological Institute (KNMI) has slightly different formula for calculating Makkink reference evaporation than currently implemented in pyet. Note that this method is only applicable to the Netherlands (~sea level). Calculating the Makkink evaporation with the formula from Makkink (1957) is more suitable for general purposes.

### CODE REVIEW

**Comment by martinvonk** [comment]
If you want I can add this method to `calculate_all` function such that it is tested. However, maybe you want to test is seperately since you already have regular `makkink`.

File: pyet/radiation.py [hunk 1]
```diff
@@ -346,6 +346,57 @@ def makkink(tmean, rs, pressure=None, elevation=None, k=0.65, clip_zero=True):
...
+    --------
+    >>> pet_mak = makkink_knmi(tmean, rs)
+
+    Notes
+    ----
+
+    This method is only applicable to the Netherlands (~sea level). Calculating
+    the Makkink evaporation with :cite:t:`makkink_testing_1957` formula is more
+    suitable for general purposes.
+
```
Comment (raoulcollenteur) [inline pyet/radiation.py:375]: Could you perhaps state what the difference is? Some empirical value ?

**Review by raoulcollenteur — CHANGES_REQUESTED** [review]

File: pyet/radiation.py (hunk 1 above)
Comment (martinvonk) [reply pyet/radiation.py]: Updated the notes.

**Comment by mvremec** [comment]
Thank you for this implementation Martin, and welcome to the PyEt family!

**Pull request merged**